cpu/native/include/cpu_conf.h

```diff
@@ -55,6 +55,11 @@ extern "C" {
  */
 #define NATIVE_ETH_PROTO 0x1234
 
+#if GNRC_PKTBUF_SIZE < 2048
+#   undef  GNRC_PKTBUF_SIZE
+#   define GNRC_PKTBUF_SIZE     (2048)
```

Comment:
> where native needs a totaly valid exception

And why don't you want to make this exception in native itself instead of the examples?